### DISCUSSION TITLE
Aeotec ZW175 becomes unavailable after update from 1.2 to 1.3

### DISCUSSION THREAD

**Discussion by andarotajo** [zwave-js/zwave-js#7143] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

**What is happening?**
Updating my Aeotec ZW175 switch from 1.2 to 1.3 makes the device become unavailable after successfully updating. The only working solution is completely removing the device and repairing it.

**What did you expect to happen instead?**
I expect the device to re-pair successfully after the update

**Steps to reproduce the behavior:**
  1. Click on update when offered to update the Aeotec ZW175 to 1.3
  2. Wait for update to complete

**Anything else we should know?**
Tried re-interviewing. This is the third switch I tried with the same result. Log file contains complete update process and re-interview attempt until the device becomes unavailable

### Software versions

**Driver (node-zwave-js):** 12.11.2

**Home Assistant Z-Wave Integration:** 2024.8.3

**Home Assistant Z-Wave JS Addon:** 0.6.2

### Device information

**Manufacturer:** Aeotec
**Model name:** ZW175-C
**Node ID:** 10

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [x] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[home-assistant_zwave_js_2024-09-02T20-11-47.706Z.log](https://github.com/user-attachments/files/16841100/home-assistant_zwave_js_2024-09-02T20-11-47.706Z.log)

**Comment by AlCalzone** [comment]
@Aeotec-ccheng is this known? The [support page](https://aeotec.freshdesk.com/support/solutions/articles/6000270359-update-smart-switch-7-eu-v1-03-zw175-c-) does not seem to mention it.

If so, it should be mentioned in the changelog of the firmware update file at least - we don't have any other way to indicate breaking changes like these yet.

**Comment by Aeotec-ccheng** [comment]
In some of our devices, firmware updating a product may cause the device to factory reset. I am uncertain if this is one of those products that this happens to.

Usually this happens if the firmware changes are great enough between different firmware Z-Wave SDK used. 

I'm not sure about this particular device since its an EU product only which i don't currently have. 

Based on details, very likely, after firmware updating, the device factory reset itself when it finished updating.

**Reply by AlCalzone** [reply]
From the logs I've seen, the device was still included and communicating but it seems the security keys at least have been reset, so the commands are not understood.

**Reply by mundschenk-at** [reply]
While I haven't looked at the logs, nodes responded to pings after the FW update but were unable to finish a re-interview.  experimented with factory reset without exclusion, but that didn't help. Only a full factory reset with exclusion made the switch work again. Unfortunately I was never able to replace the nodes "in place" while using Smart Start, so every ZW175 FW update leads to a "skipped" node number.
